### PR TITLE
bug: Fix page title from not found

### LINKED ISSUE #5041: Title in Admin always says "Not-found"
## Issue Description
Title in admin always says "Not-found". Sort of a bummer if you want to bookmark things

## Steps to Reproduce
1. Go to the admin
1. Click around
1. Observe it always says "Not-found" in the tab title

### Versions
RC10

### PR DESCRIPTION
**Pull request by bt3gl**
Resolves #5041   
Impact: **minor**  
Type: **bugfix**

## Issue
Title in admin pages was saying "Not-found" every tab. 

## Solution
Pairing with @mikemurray, this solution had two main steps:
1) We added an if for the "not-found" title to display "Reaction".
2) We added shop name with Helmet in the dashboard.

Screen Shot 2019-06-27 at 4 27 33 PM
Screen Shot 2019-06-27 at 4 27 25 PM
Screen Shot 2019-06-27 at 4 27 18 PM
Screen Shot 2019-06-27 at 4 27 13 PM

## Breaking changes
None.

## Testing
1. Load Reaction and click on each tab.

### CODE REVIEW

**Comment by willopez** [comment]
FYI: @bt3gl there are lint errors on this PR: https://circleci.com/gh/reactioncommerce/reaction/46370?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link

File: imports/plugins/core/dashboard/client/components/OperatorLanding.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ import ShopLogoWithData from "/imports/client/ui/components/ShopLogoWithData/Sho
 function OperatorLanding() {
   return (
     <Fragment>
-      <Helmet title="Operator UI" />
+      <Helmet title="Reaction" />
```
Comment (zenweasel) [inline imports/plugins/core/dashboard/client/components/OperatorLanding.js]: What's the reasoning behind hard-coding in "Reaction" here

Comment (willopez) [reply imports/plugins/core/dashboard/client/components/OperatorLanding.js]: Hard coded value has been removed and replaced with a default value.

**Comment by zenweasel** [comment]
@bt3gl In the future if @manueldelreal is tagged on a PR let's wait for his approval before merging. We don't want to merge in PR's that haven't passed QA.

**Comment by bt3gl** [comment]
@zenweasel, sounds good and will do. In this specific case, I merged after chatting with @willopez.

**Comment by zenweasel** [comment]
@bt3gl Yeah, regardless. No one has the authority to bypass QA

**Pull request merged**